src/services/OrderService.ts

```diff
@@ -8,6 +8,14 @@ const getOpenOrders = async (payload: any): Promise <any> => {
   });
 }
 
+const getOpenOrder = async (payload: any): Promise <any> => {
```

Comment:
I think we can name it something like `getOrderDetails`, WDYT @adityasharma7 Sir?

Reply:
Makes sense

Reply:
Changed service name.